lib/ansible/module_utils/oneview.py

```diff
@@ -0,0 +1,782 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import,
+                        division,
+                        print_function,
+                        unicode_literals)
+
+try:
+    from future import standard_library
+    HAS_FUTURE = True
+except:
+    HAS_FUTURE = False
+    pass
+import json
+import logging
+import os
+import collections
+from ansible.module_utils.basic import AnsibleModule
+from copy import deepcopy
+
+if HAS_FUTURE:
+    standard_library.install_aliases()
+
+logger = logging.getLogger(__name__)
```

Comment:
We usually use the logging facilities built into AnsibleModule instead of the stdlib's logging module.
```
module = AnsibleModule([...])
module.log()
```